UI Altercations

JacobYankee opened the pull request:
Fix: UI Altercations

Theme is now dependent on device theme. The text on an email summary category was slightly changed for consistency. The volume page functionality is now in the settings page and not on a separate page.

* Altered settings page
* Theme is dependent on device theme
* Changed button text

Addresses #60

JacobYankee committed:
UI Altercations

Fix: UI Altercations

Theme is now dependent on device theme. The text on an email summary category was slightly changed for consistency. The volume page functionality is now in the settings page and not on a separate page.

* Altered settings page
* Theme is dependent on device theme
* Changed button text

Addresses #60

ghost commented:
all code compiles.


dark mod extant
![image](https://github.com/COSC481W-2024Winter/JARVIS/assets/49133270/e2095cd6-90fa-4d30-8bd9-fbe3dc98a6a1)

this page is a little scuffed
![image](https://github.com/COSC481W-2024Winter/JARVIS/assets/49133270/b9bb3be5-d222-4788-ac73-665ee1b9d6e0)


everything else seems to function.


![image](https://github.com/COSC481W-2024Winter/JARVIS/assets/49133270/cdecc457-9746-4dcb-9eaf-a4e18fa83df4)
slightly scuffed in darkmode.

Everything looks good in lightmode.

JacobYankee committed:
UI Alteration continued

Fix: Made additional changes to UI

Google sign in button is now larger to avoid text clipping with Google logo. Email summary categories have new names on their buttons.

* Altered sign in button
* Altered email category labels

Addresses #60

emarron reviewed (approved):
looks good to me, approved.
